tests/console/suseconnect_scc.pm

```diff
@@ -41,6 +41,8 @@ sub run {
     if (is_transactional) {
         trup_call('register' . $cmd);
         trup_call('--continue run zypper --gpg-auto-import-keys refresh') if is_staging;
+        # Currently only micro6.0 support SL-Micro-Extras
+        trup_call('register -p SL-Micro-Extras/' . get_var('VERSION') . '/' . get_var('ARCH')) if (get_var('VERSION') eq '6.0');
```

Comment:
We should use function `add_suseconnect_product`, which doesn't have transactional support yet. We need add transactional support to `add_suseconnect_product` . 

I'm not also sure where enable extra repo. If it is for us, we should do it only in `install_ltp`, but others maybe also interested.  I think we should discuss it with @jlausuch when he is back from vacation.

Reply:
@jlausuch  are u ok for enable Micro Extention in suseconnect_scc?